pod/perldelta.pod

```diff
@@ -45,6 +45,17 @@ XXX For a release on a stable branch, this section aspires to be:
 
 [ List each incompatible change as a =head2 entry ]
 
+=head2 Apostrophe is no longer recognized as a global name separator
+
+The apostrophe C<'> has been recognized as a separator between
+components of package names and package variables since Perl 4, acting
+exactly like C<::> which was introduced in Perl 5.
```

Comment:
I'd put a comma before "which" here because it introduces a relative clause that just provides additional information (it's not like there are multiple versions of `::` and we are specifically talking about the one introduced in Perl 5).